Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.cxx

```diff
@@ -1411,7 +1411,7 @@ void qSlicerMarkupsModuleWidget::onDeleteMarkupPushButtonClicked(bool confirm /*
     rows << row;
     }
   // sort the list
-  qSort(rows);
```

Comment:
Could you address these in a separate pull request and review the remaining occurences as well:

```
Modules/Loadable/Annotations/Widgets/qMRMLAnnotationTreeView.cxx
390:  qSort(selected.begin(),selected.end());

Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.cxx
1513:  qSort(rows);
2032:  qSort(rows);
2237:  qSort(rows);

Libs/MRML/Widgets/qMRMLTableModel.cxx
651:  qSort(mrmlIndexList.begin(), mrmlIndexList.end(), qGreater<int>());

```

Reply:
This is now in Slicer/Slicer#5557